tests/ui/resolve/issue-105069.rs

```diff
@@ -1,5 +1,7 @@
+// check-pass
+
 use self::A::*;
-use V; //~ ERROR `V` is ambiguous
+use V;
```

Comment:
It is point to ﻿`A::V`, I am concerned that it undermines the principle that order is irrelevant